Bindings/Packages/com.virtualmaker.bindings/Runtime/Bindings.UIElements.cs

```diff
@@ -314,6 +314,24 @@ private void BindIntegerField(IntegerField intField, Property<int> property, boo
                 On<ChangeEvent<int>>(intField, e => property.Value = e.newValue);
             }
         }
+
+        public void BindField<T>(string name, Property<T> property, bool twoWay)
```

Comment:
if it's available in the same version as BindIntegerField, let's remove BindIntegerField